presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -1174,8 +1182,20 @@ private boolean insertOperationsSupported(HiveStorageFormat storageFormat)
         return storageFormat != HiveStorageFormat.DWRF;
     }
 
+    private Type getMatchedType(Type type)
```

Comment:
I had a quick discussion with @electrum. Let's call this method `canonicalizeType`.